component/navigation/navigation.css

```diff
@@ -0,0 +1,110 @@
+* {

```

Comment:
add root file in - index.css  then import link - import url(././.../indexfile_path)

Reply:
ok will do that